Index: Vinsany.toml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Vinsany.toml b/pyproject.toml
rename from Vinsany.toml
rename to pyproject.toml
--- a/Vinsany.toml	(revision d7dbab074ddac97f6f6a946054d29cbb3a5d590b)
+++ b/pyproject.toml	(date 1753069093995)
@@ -11,12 +11,11 @@
     "Programming Language :: Python :: 3",
     "Operating System :: OS Independent",
 ]
-license = "GPL3"
+license = "MIT"
 license-files = ["LICENSE*"]
 
 [project.urls]
-Homepage = "https://github.com/pypa/sampleproject"
-Issues = "https://github.com/pypa/sampleproject/issues"
-
+Homepage = "https://github.com/bilibili-dideng/Vinsany"
+Issues = "https://github.com/bilibili-dideng/Vinsany/issues"
 
 
